src/solidity/PrivacyInterface.sol

```diff
@@ -1,16 +1,19 @@
 pragma solidity ^0.5.9;
 
 interface PrivacyInterface {
+  function addParticipants(bytes32[] calldata participants) external returns (bool);
```

Comment:
I've changed the parameter names to make it a bit more obvious what the parameter is. You could have a look at the DefaultOnChainPrivacyGroupManagementContract.sol ..
I guess the names don't really have to be the same, but I think that could help ...

Reply:
updated